mobsf/StaticAnalyzer/views/android/converter.py

```diff
@@ -90,8 +92,11 @@ def apk_2_java(app_path, app_dir, tools_dir):
             app_path,
         ]
         fnull = open(os.devnull, 'w')
-        subprocess.call(args,
-                        stdout=fnull,
-                        stderr=subprocess.STDOUT)
+        subprocess.run(args,
+                       stdout=fnull,
+                       stderr=subprocess.STDOUT,
+                       timeout=get_jadx_timeout_value())
+    except subprocess.TimeoutExpired as timeout_expired:
+        raise JadxTimoutExpired("Timeout for jadx expired")
```

Comment:
Can we not raise an exception here? We still want to continue analysis even when jadx decompiling times out. A `logger.warning('Timeout for jadx expired')` will be sufficient here.

Reply:
Edited, thanks.